tests/unit/conditionals/third-party/elementor-activated-conditional-test.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace Yoast\WP\SEO\Tests\Unit\Conditionals\Third_Party;
+
+use Yoast\WP\SEO\Tests\Unit\TestCase;
+
+use Yoast\WP\SEO\Conditionals\Third_Party\Elementor_Activated_Conditional;
+
+
+/**
+ * Class Elementor_Activated_Conditional_Test.
+ *
+ * @group conditionals/third-party
```

Comment:
I think we shouldn't namespace groups. Just make two of them: conditionals and third-party

Reply:
Ah, you just found a bug in the unit test generator 😅 .

It creates `@group` statements based on the folder structure, which currently fails when the folders are nested.

Reply:
I have fixed it manually here!